smarts/env/hiway_env.py

```diff
@@ -69,6 +69,7 @@ def __init__(
         self,
         scenarios: Sequence[str],
         agent_specs,
+        ordered_scenarios=False,
```

Comment:
How about rename this parameter to `shuffle_scenarios` and default to `True` since `ordered` implies some sort of inherent ordering over scenarios.

Reply:
Renamed `ordered_scenarios` to `shuffle_scenarios`.